configs/fs/cluster/client.yaml

```diff
@@ -1,7 +1,21 @@
+container_image: opencurvedocker/curvefs:latest
+mdsOpt.rpcRetryOpt.addrs: 10.0.1.1:6700,10.0.1.2:6700,10.0.1.3:6700
+log_dir: /home/curve/curvefs/logs/client
+
+### if fstype is s3
 s3.ak: <>
 s3.sk: <>
 s3.endpoint: nos-eastchina1.126.net
 s3.bucket_name: curvefs
-container_image: opencurvedocker/curvefs:latest
-mdsOpt.rpcRetryOpt.addrs: 10.0.1.1:6700,10.0.1.2:6700,10.0.1.3:6700
-log_dir: /home/curve/curvefs/logs/client
\ No newline at end of file
+
+### if fstype is volume
+# curvebs cluster's mds ip and port
+curvebs.cluster: <>
+# volume size in bytes
+volumeSize: 0
```

Comment:
volumeSize default is 0? If not, give the reasonable value.

Reply:
fix